plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/get_user_information/action.py

```diff
@@ -0,0 +1,24 @@
+import insightconnect_plugin_runtime
+from .schema import GetUserInformationInput, GetUserInformationOutput, Input, Output, Component
+
+# Custom imports below
+from komand_rapid7_insightidr.util.endpoints import Users
+from komand_rapid7_insightidr.util.resource_helper import ResourceHelper
+
+
+class GetUserInformation(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="get_user_information",
+            description=Component.DESCRIPTION,
+            input=GetUserInformationInput(),
+            output=GetUserInformationOutput(),
+        )
+
+    def run(self, params={}):
+        user_rrn = params.get(Input.USER_RRN)
+        self.connection.session.headers["Accept-version"] = "strong-force-preview"
+        request = ResourceHelper(self.connection.session, self.logger)
+        self.logger.info(f"Getting the user information for {user_rrn}...")
+        response = request.make_request(Users.get_user_information(self.connection.url, user_rrn), "get")
+        return {Output.USER: response, Output.SUCCESS: True}
```

Comment:
I am wondering if thats necessary to return the `Output.SUCCESS` while in the InsightConnect there is the same variable that returns if the action passed or not 😄

Reply:
I had done it that was as I was following the other actions in that repo - https://github.com/rapid7/insightconnect-plugins/blob/db4ecd2117b9791baadcff023d808251e9f07273/plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/get_attachment_information/action.py#L25

example of response returned below - I can remove this if needed as the results will still be very similar 
![image](https://github.com/rapid7/insightconnect-plugins/assets/144030336/abbde14e-a16f-418c-a637-a9d257f3e455)